audformat/core/testing.py

```diff
@@ -217,6 +217,7 @@ def create_db(minimal: bool = False) -> Database:
         return db
 
     db.description = 'A database for unit testing.'
+    db.author = 'audEERING GmbH'
```

Comment:
Should we use our names here instead?

Reply:
Yes, I think that makes more sense. If we want to have the option to store the company name as well, I suggest to introduce another field like `organization`.

Reply:
OK, I switched to real names and created https://github.com/audeering/audformat/pull/41 so we can decide there if we want to add organization as well.